x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/getTimezoneOffsetInMs.test.ts

```diff
@@ -34,7 +34,10 @@ describe.skip('getTimezoneOffsetInMs', () => {
     });
 
     it('returns the time in milliseconds', () => {
-      expect(getTimezoneOffsetInMs(Date.now())).toEqual(21600000);
+      const now = Date.now();
+      const expectedOffset =
+        moment.tz.zone('America/Denver')!.parse(now) * 60000;
+      expect(getTimezoneOffsetInMs(Date.now())).toEqual(expectedOffset);
```

Comment:
Maybe you can add a comment about how this fixes the timezone issue. Also so we don't simplify this later, because we forget about the fix.

Reply:
Done.